test/pr-review.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -xe
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+usage() {
+  echo "Usage: $0 PR_NUMBER" 1>&2;
+  exit 1;
+}
+
+if [ -z "${PULL_NUMBER}" ]; then
+  if [ -z "$1" ];then
+    usage
+  fi
+  PR_NUMBER=$1
+fi
+
+if [ -z "${VERIFICATION_PAUSE}" ]; then
```

Comment:
Hmm I ran into this in the other script but not this one, will fix the same way.